rebar.config

```diff
@@ -65,3 +65,14 @@
     {includes, ["src"]},
     {preprocess, true}
 ]}.
+
+{hex, [
+    {doc, #{provider => edoc}},
+    {repos, [
+        #{ name => <<"hexpm">>,
+           api_url => <<"http://localhost:4000/api">>,
+           repo_url => <<"http://localhost:4000">>,
+           repo_verify => false,
+           repo_verify_origin => false}
+    ]}
+]}.
```

Comment:
^ What is the meaning of this?

Reply:
It was an unfortunate push.

Reply:
👍 at least I had the chance of dramatically asking "What is the meaning of this?"